src/client/app/translations/data.ts

```diff
@@ -1270,96 +1273,96 @@ const LocaleTranslationData = {
 		"meter.failed.to.create.meter": "No se pudo crear un medidor con mensaje: ",
 		"meter.failed.to.edit.meter": "No se pudo editar un medidor con mensaje: ",
-		"meter.hidden": "Al menos un medidor no está visible para ti.",
+		"meter.hidden": "Al menos un medidor no es visible para tí.",
 		"meter.id": "ID del medidor",
-		"meter.input.error": "Entrada no válida entonces medidor no se creó ni se editó.",
+		"meter.input.error": "Entrada no válida, por tanto no se creó o editó el medidor.",
 		"meter.unit.change.requires": "necesita cambiarse antes de cambiar el tipo de esta unidad",
 		"meter.unitName": "Unidad:",
-		"meter.url": "URL del medidor",
+		"meter.url": "URL:",
 		"meter.is.displayable": "El medidor es visualizable",
-		"meter.is.enabled": "Actualizaciones Habilitado",
-		"meter.is.not.displayable": "El medidor es visualizable",
-		"meter.is.not.enabled": "El medidor no está activada",
-		"meter.unit.is.not.editable": "This meter's unit cannot be changed and was put back to the original value because: \u{26A1}",
-		"meter.previousEnd": "Marca de tiempo de fin previo:",
+		"meter.is.enabled": "Actualizaciones activadas",
+		"meter.is.not.displayable": "El medidor no es visualizable",
+		"meter.is.not.enabled": "El medidor no está activado",
+		"meter.unit.is.not.editable": "La unidad de este medidor no puede cambiarse y se mantuvo el valor original porque: ",
+		"meter.previousEnd": "Marca de tiempo del final anterior:",
 		"meter.reading": "Lectura:",
-		"meter.readingDuplication": "Duplicación de Lectura:",
-		"meter.readingFrequency": "Frecuencia de Lectura",
-		"meter.readingGap": "Brecha de Lectura:",
-		"meter.readingVariation": "Variación de Lectura:",
-		"meter.startTimeStamp": "Marca de tiempo de inicio:",
-		"meter.successfully.create.meter": "Se creó un medidor con éxito.",
-		"meter.successfully.edited.meter": "Se editó medidor con éxito.",
+		"meter.readingDuplication": "Duplicación de lectura:",
+		"meter.readingFrequency": "Frecuencia de lectura",
+		"meter.readingGap": "Longitud entre las lecturas:",
+		"meter.readingVariation": "Variación de lecturas:",
+		"meter.startTimeStamp": "Marca de tiempo al inicio:",
+		"meter.successfully.create.meter": "Éxito al crear el medidor.",
+		"meter.successfully.edited.meter": "Éxito al editar el medidor.",
 		"meter.timeSort": "Ordenar por tiempo:",
-		"meter.time.zone": "Zona horaria del medidor",
-		"meter.type": "Tipo de medidor",
+		"meter.time.zone": "Zona horaria:",
+		"meter.type": "Tipo:",
 		"minute": "Minuto",
 		"min": "mínimo",
 		"more.energy": "más energía",
 		"name": "Nombre:",
 		"navigation": "Navegación",
-		"need.more.points": "Nececitar más puntos",
+		"need.more.points": "Nececita más puntos",
 		"no": "no",
 		"note": "Nota:",
-		"no.data.in.range": "No Data In Date Range\u{26A1}",
-		"oed": "Panel de Energía Abierta",
+		"no.data.in.range": "No hay datos para el rango de fechas",
+		"oed": "Open Energy Dashboard",
```

Comment:
I'm wondering if this should stay in English or be translated. It is a name but it also has a meaning as words. What do you think?

Reply:
I go back and forth on this myself, as well as what the best translation would be. Torn between "Panel" and "Tablero". For either one you have a funny acronym, TEA or PEA

Reply:
I decided to see how it is used to see if that would help. And....I cannot see any use of the key "oed" (or 'oed'). If it was to be used then probably src/client/index.html is the place. I tired to get that title to show but could not even by disabling JS. I think we can delete this key.